Update README (#1)
OpenTermsArchive/demo-versions: README.md

# Open Terms Archive - demo versions

## Context

> [Open Terms Archive](https://opentermsarchive.org) is free and open source software that publicly records every version of the terms of digital services in different languages and countries several times a day, increasing their readability and highlighting their changes to enable democratic oversight. Check out [opentermsarchive.org for more details](https://opentermsarchive.org).

Reviewer: In order to keep it shorter, i suggest to remove this precisions at this stage, but it's not blocking.
```suggestion
[Open Terms Archive](https://opentermsarchive.org) is free and open source software that publicly records every version of the terms of digital services, increasing their readability and highlighting their changes to enable democratic oversight. Check out [opentermsarchive.org for more details](https://opentermsarchive.org).
```

In the context of Open Terms Archive:

> - a **version** is a record of a service's terms captured at a particular point in time. These versions are derived from online documents containing the terms. The process involves extracting and filtering relevant content from these documents, allowing the system to identify any changes that may have occurred in the terms and to record a new version if any changes are detected.

Reviewer: ```suggestion
- a **version** is a record of a service's terms cleaned of the non-significant parts (menus, ads,...) of the online document.
```

> Each version represents the state of the terms at the time of its capture, and they are stored to provide a historical record of how the terms have evolved over time.

Reviewer: ```suggestion
Each version represents the state of the terms at the time of its capture, and they are publicly stored to provide a historical record of how the terms have evolved over time.
```

> - a **collection** refers to a grouping of terms tracked by Open Terms Archive. Each collection is characterised by a scope across dimensions such as language, jurisdiction and industry.

Reviewer: ```suggestion
- a **collection** refers to a grouping of terms tracked by Open Terms Archive characterised through a three-dimensional scope defined by language, jurisdiction and industry.
```

> Information about this "demo" collection, such as scope, maintener, time frame, … is detailed in [its definition repository](https://github.com/OpenTermsArchive/demo-declarations).

Reviewer: ```suggestion
Detailed information about this collection can be found at [its definition repository](https://github.com/OpenTermsArchive/demo-declarations).
```

> **This repository has to be considered as the database of versions for this collection.**

Reviewer: So what? 😅

## Usage

🔍 [Browse through versions](https://docs.opentermsarchive.org/how-to/navigate-history/) interactively on this user interface.

> 🔔 [Subscribe to terms changes](https://docs.opentermsarchive.org/how-to/navigate-history/) to be notified.

Reviewer: ```suggestion
🔔 [Subscribe by RSS to terms changes](https://docs.opentermsarchive.org/how-to/subscribe-rss/) to be notified.
```

📥 [Download versions as dataset](https://github.com/OpenTermsArchive/demo-versions/releases) to explore data locally on your filesystem.

> If a specific service or specific terms for a service is missing from this collection, it may be available in others [public Open Terms Archive collections](https://github.com/OpenTermsArchive?q=versions&type=all&language=&sort=), and if not, [it can be added](https://docs.opentermsarchive.org/contributing-terms).

Reviewer: ```suggestion
If a specific service or specific terms for a service is missing from this collection, it may be available in others [referenced Open Terms Archive collections](https://opentermsarchive.org/#collections). If not, it can be added by following our documentation about [contributing terms](https://docs.opentermsarchive.org/contributing-terms).
```
